wire/message.go

```diff
@@ -24,11 +24,11 @@ const CommandSize = 12
 
 // ebs is the excessive block size, used to determine reasonable maximum message sizes.
 // 32MB is the current default value
-var ebs uint32 = 32000000
+var ebs uint32 = 32000000 * 4
```

Comment:
Yeah this * 4 is all over the place, where'd we get this and should we set a constant somewhere?

Reply:
Commented above. https://github.com/gcash/bchd/pull/545#discussion_r1842667454